### ISSUE TITLE
Uncaught SyntaxError: Unexpected token { (WARNING: non-Error used)

### ISSUE THREAD

**Issue by chenxsan** [geelen/jspm-loader-css#15]
With the latest babel:

```
System.config({
  "map": {
    "babel": "npm:babel-core@5.8.20",
    "babel-runtime": "npm:babel-runtime@5.8.20",
```

`import '../style/app.css!` with jspm-loader-css would throw errors in console:

<img width="1271" alt="screen shot 2015-08-02 at 21 41 51" src="https://cloud.githubusercontent.com/assets/1091472/9026362/538894a6-395f-11e5-8c79-e67777a3328e.png">

I tried jspm-loader-css with 

```
"babel": "npm:babel-core@5.7.4",
    "babel-runtime": "npm:babel-runtime@5.7.0",
```

Everything works as expected.

**Comment by chenxsan** [comment]
I thought it maybe the problem of babel, just tested babel@5.8.19, no errors then.

**Comment by chenxsan** [comment]
Just in case someone else happened to see this error, I would like to write here what I found.

1. babel-core@5.8.20 & babel-core@5.8.21 would cause errors when using jspm-loader-css, I tested them with https://github.com/css-modules/jspm-demo project. So I use babel-core@5.8.19 for the time being.
2. I happened to install babel with `npm install babel --save-dev`, seems this would cause errors too.

If none solves the problem, try to remove the whole jspm_packages folder, and run `jspm install` again.
